### PR TITLE
Bump Shopify API and tap-singer versions

### PR DESCRIPTION
**Pull request by ghost**
## What

As described in https://github.com/singer-io/tap-shopify/pull/79:

> For marketing purposes, it's very useful to have the landing_site field in Orders, which contains UTM parameters.
> 
> These are only available in Shopify API version 2021-01. Support for this seems to have been added in ShopifyAPI 8.2.0 (https://github.com/Shopify/shopify_python_api/blob/60d6c23244a8641dfd25983d4206f49bb1d15f50/CHANGELOG#L3-L5).

## How

Bump Shopify API in upstream Singer tap, bump Singer tap here.

## Remarks

This is a draft PR as it depends on an unmerged patch to upstream.

Using this to get feedback on what else, if anything, needs changing to bump an Airbyte connector.  

## Pre-merge Checklist

I ran the local tests I could find (gradlew and friends), and successfully sync'd my shop's data with this patch.

- [ ] *Run integration tests*
- [ ] *Publish Docker images*

### CODE REVIEW

**Comment by sherifnada** [comment]
@ns-admetrics Thanks for creating the PR! ideally we can bump the singer tap a version once the upstream accepts the change. If this is not possible or takes too long, we're happy to create our own fork of this tap to pick up the version. How blocking is this for you?

**Comment by ns-admetrics** [comment]
Well, on the one hand I can run my own container builds :) On the other, it seems upstream is not super responsive, and would have to go through a release process beside merging. So for faster iteration, it might not be wrong to focus on developing in your repo. On a side note; what if anything would be useful to change/fix to turn the Shopify source into a more "native" Airbyte source?

**Comment by sherifnada** [comment]
@ns-admetrics we'd definitely love to create a native Shopify connector for exactly the reasons you mentioned plus it would make it easier to conform the connector to our best practices. If this is something you're interested in working on, we'd love to support you during it. Otherwise we're happy to add it to our connector backlog. 

For the purposes of this PR I can push your fork under the Airbyte org to pick up this fix. Will get to it tomorrow or the day after at the latest.

**Comment by ns-admetrics** [comment]
Let me know when I should rebase this PR or whatever.

**Comment by ns-admetrics** [comment]
Rebased on the latest published tag, `v0.14.0-alpha.0`.

File: airbyte-integrations/connectors/source-shopify-singer/Dockerfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ ENV AIRBYTE_IMPL_PATH="SourceShopifySinger"
 WORKDIR /airbyte/integration_code
 COPY $CODE_PATH ./$CODE_PATH
 COPY setup.py ./
-RUN pip install tap-shopify==1.2.6
+RUN pip install https://github.com/ns-admetrics/tap-shopify/archive/bump-shopify-api-version.zip
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-shopify-singer/Dockerfile:13]: @ns-admetrics could you instead use the tap version we forked: 
```suggestion
RUN pip install https://github.com/airbytehq/tap-shopify/tarball/master
```

File: airbyte-integrations/connectors/source-shopify-singer/Dockerfile.test [hunk 2]
```diff
@@ -18,7 +18,7 @@ COPY secrets/config.json $CODE_PATH/config.json
 COPY $MODULE_NAME/*.json $CODE_PATH/
 COPY setup.py ./
 
-RUN pip install tap-shopify==1.2.6
+RUN pip install https://github.com/ns-admetrics/tap-shopify/archive/bump-shopify-api-version.zip
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-shopify-singer/Dockerfile.test:21]:
```suggestion
RUN pip install https://github.com/airbytehq/tap-shopify/tarball/master
```

File: airbyte-integrations/connectors/source-shopify-singer/build.gradle [hunk 3]
```diff
@@ -15,6 +15,6 @@ dependencies {
 // used to allow local iteration to work
 task('installSingerTap', type: PythonTask) {
     module = "pip"
-    command = "install tap-shopify==1.2.6"
+    command = "install https://github.com/ns-admetrics/tap-shopify/archive/bump-shopify-api-version.zip"
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-shopify-singer/build.gradle:18]:
```suggestion
    command = "install https://github.com/airbytehq/tap-shopify/tarball/master"
```

**Review by sherifnada — APPROVED** [review]
@ns-admetrics LGTM -- I pulled your changes to `tap-shopify` into an [Airbyte fork](github.com/airbytehq/tap-shopify) -- if you can point to that in your deps we can publish this connector.

**Comment by ns-admetrics** [comment]
Re. merge conflict, my last rebase was onto `v0.14.0-alpha.0` as mentioned above.
Not quite sure how current `master` diverged, will there be an 0.15 release soon?
Feel free to fix yourself, "Allow edits by maintainers" is enabled.

**Pull request merged**